src/templates/Documentation.js

```diff
@@ -245,7 +245,7 @@ const Documentation = ({ data, pageContext, location }) => {
     <MDXProvider components={componentMapping}>
       <LayoutBase
         title={
-          location.pathname === "/docs"
+          normalizeRoute(location.pathname) === "/docs/"
```

Comment:
Fixes title on `/docs/`, I think trailing slashes were borking it previously in production.